tests/src/PatternAssertion/FilePatternAssert.php

```diff
@@ -46,8 +46,9 @@ protected function assertBaseElements(string $html, string $variant): void {
     $this->assertElementExists('svg.icon--2xl', $crawler);
     $this->assertElementExists('svg.icon--file', $crawler);
```

Comment:
I think we should change this, lets just check the presence of the svg, or even remove these lines (See what makes sense) as we have the IconPatternAssert test

Reply:
<img width="718" alt="image" src="https://github.com/openeuropa/oe_bootstrap_theme/assets/36895476/8e6f079a-cfdb-4222-adc1-ff94231574bc">

it's always visible inside the file but indeed we can make it 1 line.